docs/content/preview/secure/tls-encryption/server-certificates.md

```diff
@@ -322,54 +315,66 @@ Write out database with 1 new entries
 Data Base Updated
 ```
 
-{{< note title="Note" >}}
-
 Each node key and certificate should use the `node.<commonName>.[crt | key]` naming format.
 
-{{< /note >}}
-
 You can verify the signed certificate for each of the nodes by running the following `openssl verify` command:
 
 ```sh
-$ openssl verify -CAfile secure-data/ca.crt <node-ip-address>/node.<node-ip-address>.crt
+openssl verify -CAfile secure-data/ca.crt <node-ip-address>/node.<node-ip-address>.crt
 ```
 
 You should see the following output, displaying the node IP address:
 
-```
+```output
 X.X.X.X/node.X.X.X.X.crt: OK
 ```
 
+### Specify Subject Alternative Name 
+
+Recently, the `commonName` field for server certificates has been deprecated, with some browsers no longer supporting it and others ignoring the value and checking the `subjectAltName` field instead. 
+
+You can create certificates using the Subject Alternative Name (SAN) extension by following [OpenSSL x509v3_config documentation](https://www.openssl.org/docs/man1.1.1/man5/x509v3_config.html).
+
+Consider the following as a guideline:
+
+```sh
+subjectAltName=@subject_alt_section
+
+[ subject_alt_section ]
+subjectAltName=URI:ldap://somehost.com/CN=foo,OU=bar
+```
+
```

Comment:
```suggestion

```

```

Reply:
This section at the bottom for SANs can be call back since already specified in the above node configuration file

Reply:
Deleted section